docs-ref-conceptual/create-an-azure-service-principal-azure-cli.md

```diff
@@ -56,6 +56,9 @@ Record their values, but they can be retrieved at any point with [az ad sp list]
 
 For certificate-based authentication, use the `--cert` argument. This argument requires that you hold an existing certificate. Make sure any tool that uses this service principal has access to the certificate's private key. Certificates should be in an ASCII format such as PEM, CER, or DER. Pass the certificate as a string, or use the `@path` format to load the certificate from a file.
 
+> [!NOTE]
+> When using a PEM file, the **CERTIFICATE** must be appended to the **PRIVATE KEY** within the file.
```

Comment:
This statement is wrong. The private key should not be passed: https://github.com/MicrosoftDocs/azure-docs-cli/issues/4734